Assets/Mirror/Examples/AdditiveScenes/Scripts/NetworkManagerExt.cs

```diff
@@ -0,0 +1,58 @@
+﻿using UnityEngine;
+using System.Collections;
+using UnityEngine.SceneManagement;
+
+namespace Mirror.Examples.Additive
+{
+    public class NetworkManagerExt : NetworkManager
+    {
+        [Scene]
+        [Tooltip("Add all sub-scenes to this list")]
+        public string[] subScenes;
+
+        public override void OnStartServer()
+        {
+            base.OnStartServer();
+            Debug.Log("Loading Scenes");
+
+            // load all subscenes on the server only
+            foreach (string sceneName in subScenes)
+                StartCoroutine(LoadScene(sceneName));
+        }
+
+        IEnumerator LoadScene(string sceneName)
+        {
+            yield return SceneManager.LoadSceneAsync(sceneName, LoadSceneMode.Additive);
```

Comment:
yield return shouldn't be needed for loadsceneasync. are you sure that is needed?

Reply:
[Unity docs](https://docs.unity3d.com/ScriptReference/SceneManagement.SceneManager.LoadSceneAsync.html) suggest this needs to be in a coroutine, thus the yield return.